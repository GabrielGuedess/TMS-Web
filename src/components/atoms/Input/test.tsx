import { render, screen } from '@testing-library/react';

import { Input } from '.';

describe('<Input />', () => {
  it('should render the input correctly', () => {
    const { container } = render(<Input />);

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render the input with error', () => {
    render(<Input errorMessage="Error" isInvalid />);

    expect(screen.getByText('Error')).toBeInTheDocument();
  });
});
