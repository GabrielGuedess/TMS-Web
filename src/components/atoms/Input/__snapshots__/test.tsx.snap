// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should render the input correctly 1`] = `
<div
  class="relative flex h-min flex-col"
>
  <div
    class="relative"
  >
    <input
      autocomplete="on"
      class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 p-3 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
      id=":r0:"
    />
  </div>
</div>
`;
